user_guide_src/source/libraries/caching.rst

```diff
@@ -36,14 +36,14 @@ $handler
 ========
 
 The is the name of the handler that should be used as the primary handler when starting up the engine.
-Available names are: dummy, file, memcached, redis, predis, wincache.
+Available names are: dummy, file, memcached, redis, predis, wincache. This is commonly the **File** handler
+since the file system is always available, but may not fit more complex, multi-server setups.
 
 $backupHandler
 ==============
 
 In the case that the first choice ``$handler`` is not available, this is the next cache handler to load.
-This is commonly the ``File`` handler since the file system is always available, but may not fit
-more complex, multi-server setups.
+The default backup handler is **dummy**.
```

Comment:
Why do we need to add this?
Dummy is not a real cache handler. It does not cache anything.

Reply:
@kenjis $backupHandler is different in User guide and App/Config/Cache.php file. I tried to make them the same. Guide says $backupHandler is File but code has $backupHandler = 'dummy'. How should we match them? $handler = 'file', $backupHandler = 'dummy' in the code. Making two of them 'file' is a bit meaningless.

Reply:
The user guide does not say about the default backup handler.

The default `$backupHandler` should be `dummy`. Because it never fails to initialize.
And it does not cache anything, so it is the same as not using cache.

In real apps, it is often `file` and the primary cache is `memcached` or  `redis`.
Because file can be used in most environments.

Reply:
Yes. If it is a bit difficult to understand,
1. show that the default handler is dummy
2. explain that file handler is often used

Reply:
@kenjis Rolled back the backuphandler explanation.